File: examples/outbound/call_out.exs
IO.puts("double: #{inspect(:out.double([21], %{}))}")
IO.puts("greet: #{inspect(:out.greet(["elixir"], %{}))}")
p = :slop_rt.instantiate(:"out.Point", [10, 20], %{})
IO.puts("Point.sum: #{inspect(apply(:out, :"out.Point.sum", [[p], %{}, {}]))}")

File: examples/outbound/call_out.erl
-module(call_out).
-export([main/0]).
main() ->
    io:format("double: ~p~n", [out:double([21], #{})]),
    io:format("greet default: ~p~n", [out:greet([<<"beam">>], #{})]),
    Kw = #{punct => <<"?">>},
    io:format("greet kw: ~p~n", [out:greet([<<"bob">>], Kw)]),
    P = slop_rt:instantiate('out.Point', [3, 4], #{}),
    io:format("Point.sum: ~p~n", [out:'out.Point.sum'([P], #{}, {})]),
    init:stop().
